src/main/java-intellij/com/mercedesbenz/sechub/plugin/idea/sechubaccess/SecHubAccess.java

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.plugin.idea.sechubaccess;
+
+import com.mercedesbenz.sechub.api.DefaultSecHubClient;
+import com.mercedesbenz.sechub.api.SecHubClient;
+import com.mercedesbenz.sechub.api.SecHubClientException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+
+public class SecHubAccess {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SecHubAccess.class);
+    private SecHubClient client;
+    public SecHubAccess(String secHubServerUrl, String userId, String apiToken, boolean trustAllCertificates) {
+        initSecHubClient(secHubServerUrl, userId, apiToken, trustAllCertificates);
+    }
+
+    public boolean isSecHubServerAlive() {
+        if (client == null) {
+            LOG.debug("SecHub client is not initialized");
+            return false;
+        }
+        try {
+            return client.isServerAlive();
+        } catch (SecHubClientException e) {
+            return false;
+        }
+    }
+
+    private void initSecHubClient(String secHubServerUrl, String userId, String apiToken, boolean trustAllCertificates) {
+
+        SecHubClient client = null;
+
+        if (isInputMissingOrEmpty(secHubServerUrl, userId, apiToken)) {
+            this.client = client;
+            return;
```

Comment:
Since the SecHubAccess is no singleton any more I think this could be simplified.
```suggestion
        if (isInputMissingOrEmpty(secHubServerUrl, userId, apiToken)) {
            return;
```